semanticdb/metacp/src/main/scala/scala/meta/internal/javacp/Javacp.scala

```diff
@@ -236,9 +232,12 @@ object Javacp {
     directInnerClasses.foreach { ic =>
       val innerClassSymbol = ssym(ic.name)
       decls += innerClassSymbol
-      val innerPath = asmNameToPath(ic.name, toplevel.base)
-      val innerClassNode = innerPath.toClassNode
-      buf ++= sinfos(toplevel, innerClassNode, ic.access, classScope)
+      val path = ic.name + ".class"
+      val classfile = classpathIndex.getClassfile(path).getOrElse {
+        throw MissingSymbolException(path)
```

Comment:
The definition of the missing symbol exception says `final case class MissingSymbolException(symbol: String)`. I'm surprised that we're passing a path to it.

Reply:
Good catch, updated to `ssym(ic.name)`.